src/Orckestra.Composer.Website/UI.Package/Typescript/Services/AutocompleteSearchService.ts

```diff
@@ -52,6 +52,9 @@ module Orckestra.Composer {
                 this._searchCriteria.addSingleFacet(`CategoryLevel${parents.length + 1}_Facet`, suggestion);
                 this._searchCriteria.keywords = '*';
                 this._searchCriteria.correctedSearchTerm = '*';
+
+                this._eventHub.publish("autosuggestModalClosed", { data: {} });
+
                 this.search();
             }
         }
```

Comment:
I am not sure about all that changes, but is it nop enough just to override  search method inside the AutocompleteSearchService? like this:

        protected search(): void {
            const queryString = this._searchCriteria.toQuerystring();
            this._window.location.href = this._baseSearchUrl + queryString;
        }

Reply:
Also please create PR to release branch